sql/migrations/0004_next.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM proposals WHERE layer < 19000;
+DELETE FROM proposal_transactions WHERE layer < 19000;
+UPDATE certificates SET cert = NULL WHERE layer < 19000;
+UPDATE ballots SET ballot = prune_actives(ballot);
```

Comment:
considering #5002 and the fact that right now the ref ballot is still saved with activeset. 
`ActiveSetFromEpochFirstBlock` also still retrieve activeset directly from ref ballot.
is there a safe range i should apply to remove the activeset from ballots?

at layer 17800, the state size changed from 8.2GB to 470MB with this migration.

Reply:
my proposal was to save active sets into activeset table while pruning. and then we either have an option to sql-join them on activesethash field or do another query.

Reply:
> https://github.com/spacemeshos/go-spacemesh/issues/4984#issuecomment-1717409692

we should also stop writing active sets together with ballot, after this migration is done. not necessarily within same change, but would be better to release them together

Reply:
changed to extract active set from ballots and save the prune blob.

please consider https://github.com/spacemeshos/go-spacemesh/issues/4984#issuecomment-1720169998
i'll continue with the other code change accordingly.